backends-common/rabbitmq/src/main/java/org/apache/james/backends/rabbitmq/Constants.java

```diff
@@ -34,6 +36,10 @@
     String EMPTY_ROUTING_KEY = "";
     boolean REQUEUE = true;
 
+    String SINGLE_ACTIVE_CONSUMER_ARGUMENT = "x-single-active-consumer";
+    boolean SINGLE_ACTIVE_CONSUMER = true;
+    Map<String, Object> WITH_SINGLE_ACTIVE_CONSUMER = ImmutableMap.of(Constants.SINGLE_ACTIVE_CONSUMER_ARGUMENT, Constants.SINGLE_ACTIVE_CONSUMER);
```

Comment:
`Object`?

Reply:
Yes, it's the type expected in the `Channel.queueDeclare` method of the rabbitmq client for the argument ` Map<String, Object> arguments`